src/MemoryTracing/MemoryTracingUtils.cpp

```diff
@@ -21,6 +23,58 @@ namespace orbit_memory_tracing {
 using orbit_grpc_protos::kMissingInfo;
 using orbit_grpc_protos::SystemMemoryUsage;
 
+// See http://en.wikipedia.org/wiki/Kilobyte for details on SI decimal and IEC binary definitions.
+//
+// SI decimal      SI decimal   IEC binary       IEC binary
+// prefix          value        prefix           value
+// kilobyte (kB)   10^3         kibibyte (KiB)   2^10
+// megabyte (MB)   10^6         mebibyte (MiB)   2^20
+// gigabyte (GB)   10^9         gibibyte (GiB)   2^30
+// terabyte (TB)   10^12        tebibyte (TiB)   2^40
+// petabyte (PB)   10^15        pebibyte (PiB)   2^50
+// exabyte  (EB)   10^18        exbibyte (EiB)   2^60
+//
+// As we use kilobytes in the protos, the memory size unit will be converted to kilobytes.
+std::optional<float> ExtractSizeUintConversionFactor(const std::string& unit_string) {
```

Comment:
Seems not right having floating point operations here. Maybe translate to bytes? You can convert to kilobytes on the caller side. In this case it won't need any floating point results.

Reply:
Changed to convert to bytes and that really helps to get rid of the floating operations! Thanks so much for the suggestions!